modules/core/28-wasm/keeper/keeper.go

```diff
@@ -60,12 +60,10 @@ func NewKeeper(cdc codec.BinaryCodec, key sdk.StoreKey, vmConfig *VMConfig, vali
 	}
 }
 
-func (k Keeper) PushNewWASMCode(ctx sdk.Context, clientType string, code []byte) ([]byte, string, error) {
+func (k Keeper) PushNewWASMCode(ctx sdk.Context, code []byte) ([]byte, string, error) {
```

Comment:
`s/WASM/Wasm/`

Reply:
Throughout this file, in fact.

Reply:
yeah, I wanted to do it in separate commit (to not mix up things too much)